### PR TITLE
feat: add some do's to react guidelines

### PR DESCRIPTION
**Pull request by davidgg**
## Reason for change

**Guide: REACT**
After talking with @danigb about some issues we had, I'm adding a couple of extra do's/don'ts

## List of Changes

- Add function declaration 
- Add named exports
- Inline props

## Checklist

- [X] I have updated necessary documentation and links in the README.md or the doc folder
- [X] I have rebased from master, written good commit messages and squashed unnecessary commits

### CODE REVIEW

File: guides/development/react-guidelines.md [hunk 1]
```diff
@@ -72,7 +74,50 @@ We currently use:
...
+    <div>
+      <Logo />
+    </div>
+  );
+}
+```
+
+###  1.7. <a name='Donameexports'></a>Do name exports
+
+It allows to export multiple values and it encourages the use of the same naming.
```
Comment (danigb) [inline guides/development/react-guidelines.md:101]: Wow, you liked it ;-)

File: guides/development/react-guidelines.md [hunk 2]
```diff
@@ -72,7 +74,50 @@ We currently use:
...
+
+```ts
+export function LoginPage() {
+  ...
+}
+```
+
+###  1.8. <a name='Doinlinetypeprops'></a>Do inline type props
+
+When using a type once, we can save some lines declaring the type inline.
```
Comment (danigb) [inline guides/development/react-guidelines.md:111]: For me the key here is that it reduces the external dependencies of the function and therefore is more easy to move or change without errors.

**Review by danigb — COMMENTED** [review]
Glad you added those to the guidelines 👏 👏  I'd add a comment to understand better why declaring props type inline is a better idea, but I can live without it :-)

**Review by danigb — APPROVED** [review]
I wanted to approve in the first place, but I press the wrong radio button 😂

File: guides/development/react-guidelines.md (hunk 2 above)
Comment (davidgg) [reply guides/development/react-guidelines.md]: done!

File: guides/development/react-guidelines.md (hunk 1 above)
Comment (davidgg) [reply guides/development/react-guidelines.md:101]: :)

**Pull request merged**